src/main/webapp/WEB-INF/tags/user/header.tag

```diff
@@ -130,11 +112,18 @@
                                 <a <c:if test='${"/agenda.jsp" eq pageContext.request.requestURI}'> class="active"</c:if> href="/agenda">Agenda</a>
                             </li>
 
-                            <c:forEach var="tag" items="${tags}">
+                            <c:if test="${tags.size() > 0}">
```

Comment:
We use the tags not only for previous years. We use it for grouping of similar content as well. For example at the moment we have a Blog tag besides 2015, 2016, 2017 and 2018.

I'm afraid that with this change you'll put the Blog (and any other future tags) under this Previous years menu. Which is not our goal

Reply:
Can we then cahnge the menu text to somthing liek Blogs & Articles?

Reply:
That's more a hack solution. I think we need a more general one, where previous years are not simple tags.